superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -646,14 +674,14 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
       ),
       recipients,
       report_format: reportFormat || DEFAULT_NOTIFICATION_FORMAT,
+      extra: contentType === 'dashboard' ? currentAlert?.extra : null,
```

Comment:
Can we create a type for this `contentType` instead of relying on strings?

Reply:
I've added a `ContentType` Enum in the latest commit to address this.